common/protob/messages-ethereum.proto

```diff
@@ -66,7 +66,16 @@ message EthereumSignTx {
     optional bytes data_initial_chunk = 7;  // The initial data chunk (<= 1024 bytes)
     optional uint32 data_length = 8;        // Length of transaction payload
     optional uint32 chain_id = 9;           // Chain Id for EIP 155
-    optional uint32 tx_type = 10;           // (only for Wanchain)
+    optional uint32 tx_type = 10;           // (Used for Wanchain and EIP2718)
+    optional bytes max_gas_fee = 12;        // <=256 bit unsigned big endian (in wei) - EIP 1559
+    optional bytes max_priority_fee = 13;   // <=256 bit unsigned big endian (in wei) - EIP 1559
+    repeated EthereumAccessList access_list = 14; // Access List
+    optional bool eip2718 = 15;             // EIP2718
+
+    message EthereumAccessList {
+        required string address = 1;
+        repeated string storage_keys = 2;
```

Comment:
The protobuf already uses `string` for the `to` address, so I assumed that to be fine?

It also seems simpler to be able to construct access_list like so:
```
access_list=[
                    [
                        "0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae",
                        [
                            "0x0000000000000000000000000000000000000000000000000000000000000003",
                            "0x0000000000000000000000000000000000000000000000000000000000000007",
                        ],
                    ]
             ]
```
Dont mind the formatting here 😆

Reply:
We always use strings for _addresses_, because on most coins the address<->bytes conversion is not straightforward.

OTOH we use bytes for pubkeys in most cases because the conversion usually _is_ straightforward, a public key is just bytes.

For construction, `protobuf.dict_to_proto` will handle the hex conversion by itself, so i'm not sure what the issue is...?

Reply:
So are you just asking for the `storage_keys` to be bytes?

Reply:
yes

Reply:
Done!